raphtory-graphql/src/model/mod.rs

```diff
@@ -167,17 +184,20 @@ pub(crate) struct Mut(MutRoot);
 
 #[MutationFields]
 impl Mut {
+    /// Returns a plugin.
     async fn plugins<'a>(_ctx: &Context<'a>) -> MutationPlugin {
         MutationPlugin::default()
     }
 
+    /// Delete graph from a path on the server.
     // If namespace is not provided, it will be set to the current working directory.
```

Comment:
I think this point here on namespace not being included is confusing - the path is the path including the relevant namespace -- wanna also add how this fails?